settings.php

```diff
@@ -76,7 +76,10 @@ function page_optimize_settings_field_css( $args ) {
 	<fieldset>
 		<label>
 			<input type="checkbox" id="page_optimize-css" name="page_optimize-css" value="1" <?php checked( get_option( 'page_optimize-css' ) ); ?>>
-			<?php _e( 'Concatenate and minify styles', page_optimize_get_text_domain() ); ?>
+			<?php _e( 'Concatenate styles', page_optimize_get_text_domain() ); ?>
+			<p class="description">
+				<?php _e( 'CSS minification is not enabled by default. Check `PAGE_OPTIMIZE_CSS_MINIFY` constant.', page_optimize_get_text_domain() ); ?>
```

Comment:
Actually, I'd completely drop this. We have explanatory text in the `readme.txt`. Section for `Installation`.